docs/modules/ROOT/pages/tutorial-onpremise.adoc

```diff
@@ -1,32 +1,39 @@
-= On-premise example
-
-[abstract]
-CMOS is deployed to monitor an on-premise cluster with a simple Nginx web server exposed on port 8080 to provide a landing page.
-
-include::partial$tutorial.adoc[]
+= CMOS On Premise Deployment
```

Comment:
Add an overview section about CMOS here. In the second paragraph of overview say something like:
```txt
CMOS can also be deployed on a VM based infrastructure, this is achieved by effectively running the docker container
```